### PR TITLE
ISPN-13483 Create global commands for site CLI command

### PR DESCRIPTION
**Pull request by pruivo**
New option added: "--all-caches"

https://issues.redhat.com/browse/ISPN-13483

### CODE REVIEW

**Comment by pruivo** [comment]
@oraNod if you could review the docs part it would be perfect! :+1:

File: cli/src/main/resources/help/site.adoc [hunk 1]
```diff
@@ -40,6 +40,9 @@ OPTIONS
 *--cache*='CACHE_NAME'::
 Specifies a cache.
 
+*--all-caches*::
```
Comment (tristantarrant) [inline cli/src/main/resources/help/site.adoc:43]: *-a,--all-caches*::

File: cli/src/main/resources/help/site.adoc [hunk 2]
```diff
@@ -40,6 +40,9 @@ OPTIONS
 *--cache*='CACHE_NAME'::
```
Comment (tristantarrant) [inline cli/src/main/resources/help/site.adoc:40]: *-c,--cache*='CACHE_NAME'::

File: cli/src/main/resources/help/site.adoc [hunk 3]
```diff
@@ -40,6 +40,9 @@ OPTIONS
...
 Specifies a cache.
 
+*--all-caches*::
+Applies the command to all caches
+
 *--site*='SITE_NAME'::
```
Comment (tristantarrant) [inline cli/src/main/resources/help/site.adoc:46]: *-s,--site*='SITE_NAME'::

**Comment by tristantarrant** [comment]
LGTM, just some small fixes for the man page

**Comment by pruivo** [comment]
updated.

**Review by tristantarrant — APPROVED** [review]
Good to go

File: cli/src/main/resources/help/site.adoc [hunk 4]
```diff
@@ -37,10 +37,13 @@ SYNOPSIS
 
 OPTIONS
 -------
-*--cache*='CACHE_NAME'::
+*-c, --cache*='CACHE_NAME'::
 Specifies a cache.
 
-*--site*='SITE_NAME'::
+*-a, --all-caches*::
+Applies the command to all caches
```
Comment (oraNod) [inline cli/src/main/resources/help/site.adoc:44]:
```suggestion
Applies the command to all caches.
```

File: cli/src/main/resources/help/site.adoc [hunk 5]
```diff
@@ -56,6 +59,9 @@ EXAMPLES
 `site status --cache=mycache` +
 Returns the status of all backup locations for `mycache`.
 
+`site status --all-caches` +
+Returns the status of all backup locations for all caches with backups.
```
Comment (oraNod) [inline cli/src/main/resources/help/site.adoc:63]:
```suggestion
Returns the status of each backup location for all caches with backups.
```

File: documentation/src/main/asciidoc/topics/proc_cli_xsite_push.adoc [hunk 6]
```diff
@@ -15,4 +15,9 @@ Transfer cache state to backup locations.
 [//containers/default]> site push-site-state --cache=cacheName --site=NYC
 ----
 
+[TIP]
+====
+Use `--all-caches` to push state transfer for all caches.
```
Comment (oraNod) [inline documentation/src/main/asciidoc/topics/proc_cli_xsite_push.adoc:20]:
```suggestion
Use the `--all-caches` option to push state transfer for all caches.
```

File: documentation/src/main/asciidoc/topics/proc_cli_xsite_sites.adoc [hunk 7]
```diff
@@ -20,6 +20,12 @@ Take backup locations offline manually and bring them back online.
 `--site` is an optional argument. If not set, the CLI returns all backup
 locations.
 ====
+
+[TIP]
+====
+Use `--all-caches` to get the backup location status for all caches.
```
Comment (oraNod) [inline documentation/src/main/asciidoc/topics/proc_cli_xsite_sites.adoc:26]:
```suggestion
Use the `--all-caches` option to get the backup location status for all caches.
```

File: documentation/src/main/asciidoc/topics/proc_cli_xsite_sites.adoc [hunk 8]
```diff
@@ -35,4 +41,10 @@ locations.
 [//containers/default]> site take-offline --cache=customers --site=NYC
 ----
 
+[TIP]
+====
+Use `--all-caches` to bring online or take offline a backup location to all caches.
```
Comment (oraNod) [inline documentation/src/main/asciidoc/topics/proc_cli_xsite_sites.adoc:46]:
```suggestion
Use the `--all-caches` option to bring a backup location online, or take a backup location offline, for all caches.
```

File: documentation/src/main/asciidoc/topics/ref_rest_xsite_cache_managers.adoc [hunk 9]
```diff
@@ -30,6 +30,13 @@ include::json/xsite_backup_status.json[]
 | Some nodes in the local cluster have a cross-site view with the backup location, other nodes in the local cluster do not have a cross-site view. The response indicates status for each node.
 |===
 
+[source,options="nowrap",subs=attributes+]
+----
+GET /rest/v2/cache-managers/{cacheManagerName}/x-site/backups/{site}
+----
+
+Same as above but it returns the status for a single backup location.
```
Comment (oraNod) [inline documentation/src/main/asciidoc/topics/ref_rest_xsite_cache_managers.adoc:38]:
```suggestion
Returns the status for a single backup location.
```

**Review by oraNod — APPROVED** [review]
Some doc nits here @pruivo

**Comment by tristantarrant** [comment]
Merged, thanks

**Pull request merged**